tools-adoption.md

```diff
@@ -0,0 +1,23 @@
+# Tools across our projects
+
+This page is supposed to track the adoption of the tools that want to use across our projects.
+
+## Goals
+
+- Migrate from Travis CI to GitHub Actions
+- Migrate GitHub wiki documentation to Docusaurus
+- Have a deployable version of each app. E.g.: Heroku or GitHub Pages or mobile apk
+
+## Tracking
+
+| Project | GitHub Actions | Docusaurus | Heroku | GitHub Pages |
+|-|:-:|:-:|:-:|:-:|
+| anitab-org.github.io | ✅ | ❌ | N/A | ✅ |
+| mentorship-backend | ✅ | ❌ | ✅ | N/A |
+| mentorship-android | ❌ | ❌ | N/A | ✅ |
```

Comment:
@isabelcosta mentorship android does have github actions implemented . You can check the workflow [here](https://github.com/anitab-org/mentorship-android/tree/develop/.github/workflows)

Reply:
cool, I see it just does not have run tests as part of it, I think I should put this somewhere on this table 🤔